skopt/tree_opt.py

```diff
@@ -265,7 +265,7 @@ def gbrt_minimize(func, dimensions, base_estimator=None, n_calls=100,
         Control the verbosity. It is advised to set the verbosity to True
         for long optimization runs.
 
-    * `callback` [callable, optiona]
+    * `callback` [callable, optional]
         If provided, then `callback(res)` is called after call to func.
```

Comment:
the trouble with copy pasting this stuff, is that I need to copy paste it everywhere again ;)